event-loop.go

```diff
@@ -23,13 +23,17 @@ type EventLoop struct {
 
 	// timeout for non-Rendering events that needs to be processed in a polling manner
 	platformMessageRefreshRate time.Duration
+
+	// indetifer for the current thread
+	mainThreadID uint64
 }
 
 func newEventLoop(postEmptyEvent func(), onExpiredTask func(*embedder.FlutterTask) embedder.Result) *EventLoop {
 	pq := priorityqueue.NewPriorityQueue()
 	heap.Init(pq)
 	return &EventLoop{
 		priorityqueue:  pq,
+		mainThreadID:   embedder.GetCurrentTheradID(),
```

Comment:
Typo `Therad`. Also: should we maybe make this function available from an internal package, not as part of the embedder package?